runtime-modules/storage/src/lib.rs

```diff
@@ -188,6 +188,9 @@ pub trait WeightInfo {
 
 type WeightInfoStorage<T> = <T as Trait>::WeightInfo;
 
+/// Content ID length in bytes (32 bytes hash).
+pub const CID_LENGTH: usize = 32;
```

Comment:
Just note that both storage and distributor node currently use B58 representation of the multihash, which is 46 bytes:
```
$ ./bin/run leader:update-blacklist -a gW2jkz44uexQjrm5buTALDmq3yfnZpvH961enoTrfmhLVv --dev
2022-03-30 12:59:33:5933 info: Initialized runtime connection: ws://localhost:9944
2022-03-30 12:59:36:5936 info: Waiting for chain to be synced before proceeding.
2022-03-30 12:59:36:5936 info: Updating the blacklist...
2022-03-30 12:59:36:5936 info: Development mode is ON.
2022-03-30 12:59:36:5936 debug: Sending storage.updateBlacklist extrinsic...
2022-03-30 12:59:42:5942 error: Api Error: Error: Extrinsic execution error: InvalidCidLength ( Invalid CID length (must be 32 bytes))
```

Reply:
Thanks for catching this! Fixed: https://github.com/Joystream/joystream/pull/3393/commits/7efc372a721aab7cc3243a1816bed90fbbd51b5c